### ISSUE TITLE
def find_related_cols_by_content

### ISSUE THREAD

**Issue by Jsostmann** [NCATComp410/comp410_summer2020#14]
# path
    #    Starting path location to begin search
    #

**Comment by claesmk** [comment]
Which function are you implementing the starting path for?  See [this](https://github.com/orgs/NCATComp410/teams/pecan-cookies/discussions/2) post - Team Pecan Cookies is working from the files mentioned in that post.

**Comment by Jsostmann** [comment]
ohhhh okay I see now. Thanks!

**Comment by Jsostmann** [comment]
# dataframe_list
    #     List of pandas dataframe objects
    #
  [dataFrame_List](https://github.com/NCATComp410/comp410_summer2020/blob/0444454382e8f8ae965e5c4d86d2bc7253297c9b/dfstools/src/relationship_tools.py#L30)

**Comment by claesmk** [comment]
OK, this is a good start

**Comment by Jsostmann** [comment]
@claesmk 
Hey Mr. Mike, I had a quick question about ignoring files. I forked the Repo and when I try to push my changes, the `trip_logs.csv` file is too large. Would it be okay if I added `*.csv` to the `.gitignore` file?

**Comment by claesmk** [comment]
You bring up an excellent point @Jsostmann.  When you first clone the repo there are a couple of .csv files included in the repo that are used for testing purposes. 

It's not until you run the demo.py file that the remaining csv files get created.  I did not check-in those on purpose because I expected everyone to run the demo and download them, I don't want those to get check-in to the repo. 

If you ignore *.csv you will block upload of some csv files we might need to keep in the repo.  What you could do is add just the files we expect to have downloaded (airlines.csv, airports.csv, flights.csv, and trip_logs.csv)

What you should do is open a new issue for this and in that issue just commit the changes to .gitignore

**Comment by Jsostmann** [comment]
Oh okay! so your saying that inside `.gitignore` just add
 ```
...
...
airlines.csv
airports.csv
flights.csv
trip_logs.csv
```
instead of `*.csv` because these are just the test files we assume everyone has downloaded locally already?

**Comment by claesmk** [comment]
I think you will need to add the full relative path - ie 

data/trip_logs/trip_logs.csv

**Comment by Jsostmann** [comment]
Okay, Thanks for the clarification!
